src/Inkluzitron/appsettings.json

```diff
@@ -48,7 +48,7 @@
         "Self": "{0} proti sobě nemusí rollovat.",
         "InCompliance": "{0} má navrch nad {3}.\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*",
         "RollSucceeded": "{0} má štěstí proti {3}, padlo {6}/{7} (bylo potřeba {8}+).\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*",
-        "RollFailed": "{0} má proti {3} smůlu, padlo {6}/{7} (bylo potřeba {8}+).\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*\n**Tímto neúspěšným pokusem přišel {0} o {9} bodů.**"
+        "RollFailed": "{0} má proti {3} smůlu, padlo {6}/{7} (bylo potřeba {8}+).\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*\n**Tímto neúspěšným pokusem přišel {0} o {9} bodů, které dostal {3}.**"
```

Comment:
dostat je může i žena, chtělo by to zachovat neutrálnost, třeba "které byly převedeny {3}"

Reply:
Tru, ženy vlastně existují